examples/requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests>=2.7.0
 pyrestcli>=0.6.4
+configparser>=3.5.0
```

Comment:
I don't like adding a library to the main requirements just for an example. I guess this is here just to make `configparser` work with python2 and 3. In that case, you could replace it with `six` (which is already a dependency through `requests`).

Or maybe have a different requirements for examples (or use the `test_requirements`?).

Not sure what is best though.

Reply:
These are requirements for the examples, not for the global one. Even if I used `six` I'd like to have it in the `examples/requirements.txt`, cause otherwise it is counting on it as as side effect of another library (i.e calling for problems if they decided to change requirements of `requests` lib)